s/psql

```diff
@@ -32,11 +32,11 @@ case ${ENVIRONMENT} in
         ;;
 
     'staging')
-        connect_to_azure_db 'staging' 'npda_staging'
+        connect_to_azure_db 'staging' 'npda_staging' "NPDA\ Database\ Owners"
```

Comment:
Unfortunate typo in the existing staging owners group, too difficult to change now